src/coreclr/debug/di/rspriv.h

```diff
@@ -2920,10 +2920,14 @@ struct DbgAssertAppDomainDeletedData
 #ifdef OUT_OF_PROCESS_SETTHREADCONTEXT
 class UnmanagedThreadTracker
 {
-    DWORD m_dwThreadId = (DWORD)-1;
-    HANDLE m_hThread = INVALID_HANDLE_VALUE;
-    CORDB_ADDRESS_TYPE *m_pPatchSkipAddress = NULL;
-    DWORD m_dwSuspendCount = 0;
+    DWORD m_dwThreadId = (DWORD)-1; // The OS thread ID of the unmanaged thread we are tracking
+    HANDLE m_hThread = INVALID_HANDLE_VALUE; // Handle to the unmanaged thread, used for suspending and resuming the thread
+    CORDB_ADDRESS_TYPE *m_pPatchSkipAddress = NULL; // If non-NULL, this is the address to which we should set the IP when resuming the thread.
+    DWORD m_dwSuspendCount = 0; // The suspend count of the thread when we last checked it.
+    bool m_pendingSetIP = false; // Set to true if there is a breakpoint or single-step operation on target thread that doesn't have a vmTHread
```

Comment:
```suggestion
    bool m_pendingSetIP = false; // Set to true if there is a breakpoint or single-step operation on target thread
```
That vmThread part isn't accurate is it?